src/main/services/cloneFromBackupService.ts

```diff
@@ -97,12 +97,15 @@ const setupDestinationSite = async (context: BackupMachineContext) => {
 	}
 
 	const dupID = shortid.generate();
-	const dupSite = new Local.Site(baseSite);
+	const dupSite: Site = new Local.Site(baseSite);
 
 	const localSitesDir = path.dirname(baseSite.path);
 
 	dupSite.id = dupID;
 	dupSite.name = newSiteName;
+
+	delete dupSite.localBackupRepoID;
+
```

Comment:
Wow, I can't believe it came down to this! Nice find and so much easier of a fix than I was imagining (just a matter of tracking down). Would add a comment here describing why this is necessary here (for future reference).